src/internal/noscrypt_logger.hpp

```diff
@@ -10,7 +10,7 @@
 */
 #define NC_LOG_ERROR(result) _printNoscryptError(result, __func__, __LINE__)
 
-static inline void _printNoscryptError(NCResult result, const std::string funcName, int lineNum)
+static void _printNoscryptError(NCResult result, const std::string funcName, int lineNum)
```

Comment:
Because this is in an http file, I think inline is implicit if the function has a definition in the header?

Reply:
As it turned out, I also had to remove the `static` keyword.  I found out that `static` has a [different meaning](https://en.cppreference.com/w/cpp/language/storage_duration) for non-member functions, and doesn't apply to non-member functions outside of a namespace.  I think it's implicitly static, as a global-scoped function.

Reply:
Static keyword in C means the function is scoped to the translation unit it was declared in. Since we moved the function to it's own translation unit (cpp file) then the static keyword cannot be applied since it's always located externally in the object file, not in source.